doc/installationguide.md

```diff
@@ -21,21 +21,51 @@ These are the parameters that can be configured in the global section:
         port: 4041
     	}
 ```
-* **stats**: configure the periodic collection of statistics. Use `interval` in miliseconds to set the time between stats writings.
+* **stats**: configure the periodic collection of statistics. Use `interval` in milliseconds to set the time between stats writings.
 ```
     stats: {
         interval: 100
     }
 ```
-* **authentication**: authentication data, for use in retrieving tokens for devices with a trust token (just needed in scenarios with security enabled in the Context Broker side). E.g.:
-```
-	{
-        host: 'localhost',
-        port: '5000',
-        user: 'iotagent',
-        password: 'iotagent'
-	}
-```
+* **authentication**: authentication data, for use in retrieving tokens for devices with a trust token (required in scenarios with security enabled in the Context Broker side). 
+  Currently, two authentication provider are supported: `keystone` and `oauth2`. 
+  In `keystone` based authentication, the `trust` associated to the `device` or `deviceGroup` is a token representing a specific user and his rights on a given domain (i.e.
+  combination of `fiware-service` and `service-path`). The authentication process use the trust delegation workflow to check if the trust provided is valid, in which
```

Comment:
I understant that `service-path` refers to the name of the header. In that case, the right name is `fiware-servicepath`

Reply:
correct :) btw I hope my understanding of the usage of domain for auth in keystone is correct. will fix it.

Reply:
fix by 9ea2773